describeclientquotas.go

```diff
@@ -0,0 +1,128 @@
+package kafka
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/segmentio/kafka-go/protocol/describeclientquotas"
+)
+
+// DescribeClientQuotasRequest represents a request sent to a kafka broker to
+// describe client quotas.
+type DescribeClientQuotasRequest struct {
+	// Address of the kafka broker to send the request to
+	Addr net.Addr
+
+	// List of quota components to describe.
+	Components []DescribeClientQuotasRequestComponent `kafka:"min=v0,max=v1"`
```

Comment:
I don't believe these kafka tags are needed for the client level requests/responses.